msal/build.gradle

```diff
@@ -181,7 +181,7 @@ task sourcesJar(type: Jar) {
 
 // In dev, we want to keep the dependencies(common4j, broker4j, common) to 0.0.+ to be able to be consumed by daily dev pipeline.
 // In release/*, we change these to specific versions being consumed.
-String commonVersion = project.hasProperty("distCommonVersion") ? project.distCommonVersion : "0.0.+"
+String commonVersion = project.hasProperty("distCommonVersion") ? project.distCommonVersion : "7.0.1"
```

Comment:
Please revert this change. 
See comments in L182-183

Reply:
Change effected.